doc/stdlib/arrays.rst

```diff
@@ -187,7 +187,7 @@ Constructors
 
    Create an array of all zeros with the same element type and shape as ``A``\ .
 
-.. function:: ones([T::Type=Float64,] dims)
+.. function:: ones(type, dims)
```

Comment:
you can always add empty signatures outside the loop just for the purpose of attaching the docstring?